serverless.yml

```diff
@@ -38,7 +38,8 @@ functions:
       GSSO_URL: ${ssm:/repairs-hub/${self:provider.stage}/gsso-url}
       GSSO_TOKEN_NAME: ${ssm:/repairs-hub/${self:provider.stage}/gsso-token-name}
       HACKNEY_JWT_SECRET: ${ssm:/repairs-hub/${self:provider.stage}/hackney-jwt-secret}
-      HACKNEY_AUTHORISED_GROUP: ${ssm:/repairs-hub/${self:provider.stage}/authorised-group}
+      REPAIRS_AGENTS_GOOGLE_GROUPNAME: ${ssm:/repairs-hub/${self:provider.stage}/authorised-group}
```

Comment:
Another tiny change, which I am happy if it will be done separately is maybe renaming the key `authorised-group` in parameter stores in `agents-authorised-group` maybe. This will give us a clearer understanding of the group's role

Reply:
Thanks. Noted.